apps/web/src/components/Common/Layout.tsx

```diff
@@ -116,6 +122,8 @@ const Layout: FC<LayoutProps> = ({ children }) => {
         position="bottom-right"
         toastOptions={getToastOptions(resolvedTheme)}
       />
+      {showSpacesLobby ? <Spaces /> : null}
```

Comment:
nit: if `showSpacesLobby` is boolean use `&&` rather than ternary

Reply:
Updated